src/sentry/workflow_engine/models/workflow.py

```diff
@@ -36,7 +38,8 @@ class Workflow(DefaultFieldsModel, OwnerModel, JSONConfigBase):
 
     @property
     def CONFIG_SCHEMA(self) -> dict[str, Any]:
-        raise NotImplementedError('Subclasses must define a "CONFIG_SCHEMA" attribute')
+        # TODO: fill in
+        return {}
```

Comment:
Is the plan to use subclasses here? Won't we run into the same problem we proxy models?

Reply:
there isn't a type on workflow so all of them _should_ have the same config schema

Reply:
I'm not too clear on what these configs are going to be... If they're consistent for all Workflows this is fine though